x-pack/plugins/infra/public/store/remote/log_entries/operations/load.ts

```diff
@@ -22,7 +22,7 @@ export const loadEntriesActionCreators = createGraphqlOperationActionCreators<
 
```

Comment:
If I'm not mistaken this results in a prefix `x-pack/infra/remote/entries` on actions related to log entries. Can we make it `log_entries`?

![image](https://user-images.githubusercontent.com/189073/44408438-ab0d0200-a560-11e8-9b7b-062cb8411dc1.png)

Reply:
Indeed, good catch! I'll fix that and the other ones below.